uploadscript/script.js

```diff
@@ -42,14 +42,15 @@ const performUpload = async () => {
         let incorrectFolder = wordFolder + "incorrect/";
         let wordRef = db.collection("words").doc();
         // Create the base assessment information
+        const ids = [1, 2, 3, 4].sort((_a, _b) => Math.random() - 0.5)
         let wordBaseInfo = {
             value: word,
-            correctImage: await uploadFileToFirebaseStorage(correctFolder + "correct_assessment.jpg", wordRef.id + "/correct_assessment.jpg"),
+            correctImage: await uploadFileToFirebaseStorage(correctFolder + "correct_assessment.jpg", wordRef.id + `/assessment${ids[0]}.jpg`),
```

Comment:
does the first part with the correctFolder stuff also need to be changed? or is that internal / just organizing them for the researchers

Reply:
i believe this is coming from the local filesystem so retaining the existing naming convention allows us to keep distinctions between correct + incorrect, but once we upload them they will lose that distinction (by name at least)